e2e/playwright.config.ts

```diff
@@ -37,6 +38,11 @@ export default defineConfig({
 
   /* Configure projects for major browsers */
   projects: [
+    {
+      name: 'setup',
+      testMatch: 'global-setup.ts'
+    },
+
```

Comment:
Just to confirm - some of our E2E tests do test an unauthenticated page view. This configuration will not prevent that, right?

Reply:
Correct this configuration won't prevent unauthenticated page view tests. Authenticated page view tests needs to add  test.use({ storageState: 'LoginAuth.json' }) at the beginning of the test file to have the login state.

Reply:
Awesome, thank you for clarifying~!

Reply:
Great. Okay then I am resolving this conversation.